### ISSUE TITLE
failed international payment on VIVA Wallet

### ISSUE THREAD

**Issue by varungoswami** [VivaPayments/API#1203]
I have website running on Magento 1.9 and have native checkout Before we were able to get payment from Abroad, now every customer out of the UK can not process their card via Viva Wallet. I have almost 4k plus failed order, Domestic UK customer are paying without any issue.

**Comment by VivaWalletSupport** [comment]
Hello, are you using a different currency for your customers outsided the UK or do you have all orders using GBP?

**Comment by varungoswami** [comment]
Hi Tom,




Thank you for reply, all transaction done in GBP.




Thank you,




Kind regards,




Varun Goswami




Thank you,


Kind regards,


Varun Goswami


CBDVAPE4U Team


￼￼￼









On Wed, Nov 20, 2019 at 12:29 PM +0000, "Tom van der Veer" <notifications@github.com> wrote:












Hello, are you using a different currency for your customers outsided the UK or do you have all orders using GBP?

**Comment by VivaWalletSupport** [comment]
Thank you for your reply, in that case I do not see a problem with the plugin, maybe @matthewrgourd can help you further.

**Comment by varungoswami** [comment]
Hi @matthewrgourd,
I hope you are well, Could please help me out in this situation, If you require further information please let me know.
Thank you,
Varun Goswami

**Comment by matthewrgourd** [comment]
Hi @varungoswami sorry to hear you've been experiencing issues. Could you please send your Merchant ID to mgourd@vivawallet.com referencing this issue?

**Comment by matthewrgourd** [comment]
Closing issue as no reply from user.

**Comment by varungoswami** [comment]
Hi Matthew,


I have already sent an email, I have not heard back from you and I saw notification that issue was closed.
My merchant id is
691e86c8-c565-4aa4-beb8-329173fcf54b
Let me know if you required any further information, I had 6 orders cancelled on 7th and 8th December.


Thank you,
Best Regards,
Varun Goswami

On 4 December 2019 at 15:18, matthewrgourd <notifications@github.com> wrote:



Closing issue as no reply from user.
